libs/api-sdk/src/apiSdk.ts

```diff
@@ -20,7 +20,7 @@ export default class ApiSdk {
     /**
      * Initializes the ApiSdk object with a Supported URL or custom URL.
      * @param url Supported URL or custom URL.
-     * @param config Request config.
+     * @param config [Axios](https://axios-http.com/docs/req_config) request config type.
```

Comment:
Could you put `@param config [Axios](https://axios-http.com/docs/req_config) Request Config.` instead of `@param config [Axios](https://axios-http.com/docs/req_config) request config type.`?